### PR TITLE
Fixes placement of Emojis and Tweet card

### PR DESCRIPTION
**Pull request by arihantparsoya**
#40 

* Tweet Display Card placement: It should open where the Bubble is
*  Increase Tweet Card Display Time to 10 seconds

### CODE REVIEW

**Comment by arihantparsoya** [comment]
@prastut , @KumShivam

File: public/js/app/utils/linegraph.js [hunk 1]
```diff
@@ -141,12 +141,12 @@ define(["d3", "twemoji"], function(d3) {
...
                     negText.html('<span style="font-size:20px">' + twemoji.convert.fromCodePoint(negetiveEmotions[1]) + '</span> ' + percentage[0] + "%")
                         .style("left", (time) + "px")
-                        .style("top", (yCoords[0] + 80) + "px")
+                        .style("top", (yCoords[0] + 195) + "px")
                         .style('opacity', 1);
 
                     posText.html(percentage[1] + "%" + '<span style="font-size:20px">' + twemoji.convert.fromCodePoint(positiveEmotions[1]) + '</span> ')
                         .style("left", (time - 50) + "px")
-                        .style("top", (yCoords[1] + 80) + "px")
+                        .style("top", (yCoords[1] + 195) + "px")
```
Comment (prastut) [inline public/js/app/utils/linegraph.js:149]: Hardcoding would be a problem since this coordinate won't work on every device.

**Review by prastut — CHANGES_REQUESTED** [review]
Hardcoding values is a nono :/

File: public/js/app/utils/linegraph.js (hunk 1 above)
Comment (prastut) [reply public/js/app/utils/linegraph.js:149]: @parsoyaarihant Try this?
https://www.sitepoint.com/how-to-translate-from-dom-to-svg-coordinates-and-back-again/

**Pull request closed without merging**